functions/src/util.ts

```diff
@@ -0,0 +1,30 @@
+import { GeneratedPairs } from "./firestore-functions-types";
+import { Lobby } from "./firestore-types/lobby";
+
+export function generatePairs(lobbyData: Lobby): GeneratedPairs {
+  const pairs: { one: string; two: string }[] = [];
+  let pairOne, pairTwo;
+  let stalker: string | undefined;
+
+  // Check if the lobby is uneven to find stalker
+  while (lobbyData.uids.length > 1) {
+    // get random pair
+    pairOne = lobbyData.uids[Math.floor(Math.random() * lobbyData.uids.length)];
+    pairTwo = lobbyData.uids[Math.floor(Math.random() * lobbyData.uids.length)];
```

Comment:
this line can be removed because of the do while loop

Reply:
we still need the while loop because if not it will only generate one pair